src/yacs-web/src/utils.js

```diff
@@ -28,6 +28,34 @@ export const readableDate = date => {
   return `${date.getMonth() + 1}/${date.getDate()}`;
 };
 
+/**
+ * Formats a `Date` into postgres date format
+ * @param {Date} date
+ * @returns {string} string in the format `YYYY-MM-DD`
+ */
+export const standardDate = date => {
```

Comment:
I'm stil working on getting to refactoring some stuff to use `momentjs` but can also use `momentjs` here, since its already installed 😄 , just a reminder, no need to change

Reply:
:+1:

Yeah, JS dates are crazy huh